addOns/oast/src/main/java/org/zaproxy/addon/oast/ExtensionOast.java

```diff
@@ -33,11 +59,128 @@ public ExtensionOast() {
     @Override
     public void hook(ExtensionHook extensionHook) {
         super.hook(extensionHook);
+        registerOastService(new BoastService(this));
+        registerOastService(new CallbackService(this));
         extensionHook.addApiImplementor(new OastApi());
+        extensionHook.addOptionsParamSet(getParam());
+        extensionHook.addSessionListener(this);
+        getOastServices().values().forEach(t -> t.hook(extensionHook));
+        if (hasView()) {
+            extensionHook.getHookView().addStatusPanel(getOastPanel());
+            extensionHook.getHookView().addOptionPanel(getOastOptionsPanel());
+            ExtensionHelp.enableHelpKey(getOastPanel(), "ui.tabs.callbacks");
```

Comment:
It would be better to start to use the help of the add-on, I'd expect the core help to be phased out with the add-on.